app/views/submissions.scala.html

```diff
@@ -46,7 +46,13 @@ <h1>@messages("submissions.title")</h1>
 
         @for( (submission, notifications) <- submissions ){
             <tr class="table-row">
-                <td class="table-cell">@submission.ducr</td>
+                <td class="table-cell">
+                    @if(notifications.map(_.functionCode).contains(SubmissionStatus.Rejected.fullCode)) {
```

Comment:
```
notifications.map(_.functionCode).contains(SubmissionStatus.Rejected.fullCode)
```
Can we hide this structural detail under some object method ?

Reply:
Yes, done